src/odemis/gui/cont/tabs/cryo_chamber_tab.py

```diff
@@ -683,6 +714,20 @@ def _on_aligner_btn(self, evt):
         self._show_warning_msg(None)
         self.panel.btn_cancel.Enable()
 
+    def _edit_meteor_positions(self, evt):
+        """Open a dialog for editing meteor stage positions and metadata"""
+
+        stage_md = self._stage.getMetadata()
+        dialog = EditMeteorPositionsDialog(self.main_frame, stage_md)
+        dialog.Center()
+
+        _ = dialog.ShowModal()
```

Comment:
If you don't need the return value, just write:
```suggestion
        dialog.ShowModal()
```